lib/api/unittest/CDataFrameAnalysisRunnerTest.h

```diff
@@ -18,6 +18,8 @@ class CDataFrameAnalysisRunnerTest : public CppUnit::TestFixture {
     void testEstimateMemoryUsage_10();
     void testEstimateMemoryUsage_100();
     void testEstimateMemoryUsage_1000();
+    void testColumnsForWhichEmptyIsMissing_Classification();
```

Comment:
Let's stick to camel case we use this in 99.9% of all code (we should fix the other non-standard names in this suite as well, but you don't have to make that change)
```suggestion
    void testColumnsForWhichEmptyIsMissingClassification();
```

Reply:
Done.